src/components/atoms/TextButton/TextButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ButtonHTMLAttributes, DetailedHTMLProps } from "react";
+import classNames from "classnames";
+
+import "./TextButton.scss";
+
+export interface TextButtonProps
+  extends DetailedHTMLProps<
+    ButtonHTMLAttributes<HTMLButtonElement>,
+    HTMLButtonElement
+  > {
+  text: string;
+  containerClassName?: string;
+}
+
+export const TextButton: React.FC<TextButtonProps> = ({
```

Comment:
Note: not sure if this is the best naming, but it seems to reflect the purpose of this component good enough

Reply:
> Are you intending on adding any other specialised functionality to this? Because right now it's just a layer of abstraction that adds nothing, we may as well use <button> directly?

Yes, but it will be heavily reused. How do you see using `button` directly? With global styling?

> Why not just Button?

Maybe I should leave a few comments to merge it into Button(if needed), when we have it implemented properly?
The current implementation has a lot of logic inside
![image](https://user-images.githubusercontent.com/63194656/118463505-7c6be800-b708-11eb-9c98-c2a9e19165cf.png)

Reply:
> How do you see using button directly? With global styling?

What do you mean? Why would we need global styling?

Reply:
Oh.. the styles.. right: `src/components/atoms/TextButton/TextButton.scss`

Fine to keep then.

Reply:
> we may as well use <button> directly
I didn't understand this part